### PR TITLE
[POC] An operator to replicate data from DynamoDB to S3

### PR DESCRIPTION
**Pull request by milton0825**
This is a PoC Airflow operator that scans a DynamoDB table and upload data to S3.

### CODE REVIEW

**Comment by milton0825** [comment]
@jghoman @yeameen 
Here is the PoC that replicate data from a DynamoDB table to S3.

File: airflow/contrib/operators/dynamodb_to_s3.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        self.file_size = file_size
+        self.process_func = process_func
+        self.s3_bucket_name = s3_bucket_name
+        self.s3_key_prefix = s3_key_prefix
+        self.dynamodb_scan_kwargs = dynamodb_scan_kwargs
+
+    def execute(self, context):
+        dynamodb_client = AwsHook().get_client_type('dynamodb')
+        dynamodb_scan_kwargs = copy(self.dynamodb_scan_kwargs)
+        f = NamedTemporaryFile()
```
Comment (yeameen) [inline airflow/contrib/operators/dynamodb_to_s3.py]: `.json` suffix here? Same for other one.

Comment (milton0825) [reply airflow/contrib/operators/dynamodb_to_s3.py]: Added support for user to specify the suffix in https://github.com/lyft/incubator-airflow/pull/143/files#diff-98426c5c739f27ea6aa369d3be536186R52

Comment (yeameen) [reply airflow/contrib/operators/dynamodb_to_s3.py]: Ah, I see

**Pull request closed without merging**